config/galaxy.ini.sample

```diff
@@ -633,6 +638,29 @@ nglims_config_file = tool-data/nglims.yaml
 # https://FQDN/galaxy-prefix/gie_proxy for proxying
 #dynamic_proxy_prefix=gie_proxy
 
+# The Golang proxy also manages the docker containers more closely than the
+# NodeJS proxy, so is able to expose more container management related options
+
+# This attribute governs the minimum length of time between consecutive HTTP/WS
```

Comment:
`DEFAULT_CONTAINER_RUNTIME=120` is the sort of thing I want to get rid of. It's not used in every container, admins must look up documentation per-container image. This is really bad from an admin UX perspective.

RStudio doesn't support that variable because I didn't see you implement it, and there's no standard base image for GIEs (and I don't think there should be).

What's wrong with JS on the Galaxy side which polls the image?